ratpack-bearer-auth/src/main/java/st/ratpack/auth/handler/TokenScopeFilterHandler.java

```diff
@@ -26,15 +26,15 @@ public void handle(Context ctx) throws Exception {
 			if (!sharedScopes.isEmpty()) {
 				ctx.next();
 			} else {
-				sendError(ctx);
+				sendError(ctx, 403);
 			}
 		} else {
-			sendError(ctx);
+			sendError(ctx, 401);
```

Comment:
@jeff-blaisdell added 401 here instead even though this should not happen ever. we have the check though so we should be consistent